src/components/Layout/HomeContent.js

```diff
@@ -66,6 +66,10 @@ function Para({children}) {
   );
 }
 
+function Br() {
+  return <br className="hidden lg:inline" />;
+}
```

Comment:
```suggestion
function Br({breakPointPrefix = 'lg'}) {
  return <br className={`hidden ${breakPointPrefix}:inline`} />
}
```
1024px에서 애매하게 줄바꿈이 되는 경우는 위와 같이 컴포넌트 props를 설정해주고, 문제가 있는 아래 코드에서 props를 `'xl'`로 전달해주면 깔끔하게 해결될 것 같습니다.